src/config/types.ts

```diff
@@ -45,14 +45,9 @@ export interface MetricCharacterReplacement {
   replaceWithText: string;
 }
 
-export interface CompositeMember {
-  identifier: string;
-  displayName: string;
-}
-
 export interface CompositeMetric {
   name: string;
-  members: CompositeMember[];
+  members: string[];
   valueName: ValueType;
   showLowestValue: boolean;
 }
```

Comment:
Do composite metrics work for you?

Reply:
Maybe a question for @rekha86sundararajan since it was part of the original PR

Reply:
It was actually part of the PR I already merged. oops.
 
@xaner4  If you'll rebase this on master, we can go ahead and merge it up!

Reply:
Should we have a look at the CompositeMetric before doing any thing or will you just merge it in as is?

I don't think think there's any thing to rebase into master as my PR already builds upon the latest PR you merged.
But i'm no git expert at any means so maybe i'm wrong?

Reply:
Any thing left to do to get this merged @jdbranham?